docs/src/main/paradox/data-transformations/csv.md

```diff
@@ -57,6 +57,10 @@ Scala
 Java
 : @@snip (../../../../../csv/src/test/java/akka/stream/alpakka/csv/javadsl/CsvParsingTest.java) { #line-scanner }
 
+@scala[@github[Source on Github](/csv/src/test/scala/akka/stream/alpakka/csv/scaladsl/CsvParsingSpec.scala)]
+@java[@github[Source on Github](/csv/src/test/java/akka/stream/alpakka/csv/javadsl/CsvParsingTest.java)]
+
```

Comment:
This seems to be common and useful pattern - to include links to the source file of where the snippet has been taken from. We should consider adding it to the paradox itself so links to Github are added automatically: https://github.com/lightbend/paradox/issues/168

Reply:
Yes, it is a lot of failure-prone work to add those links manually.